tests/Unit/Services/AiUsageSummaryServiceTest.php

```
@@ -0,0 +1,140 @@
+                    'created_at' => '2026-05-10 00:01:00',
+                ],
+                [
+                    'customer_action_id' => 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',
+                    'feature' => 'document_summary',
+                    'status' => 'cache_hit',
+                    'model' => 'gpt-5.5-nano',
+                    'created_at' => '2026-05-10 00:02:00',
+                ],
+            ],
+            includedCredits: 10,
+            periodStart: new \DateTimeImmutable('2026-05-01 00:00:00'),
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '44', column '30'). (undefined)

(MissingImport)